src/__tests__/lib/exampleData.js

```diff
@@ -122,6 +122,7 @@ export const makeAccount = (
     email?: string,
     realm?: string,
     apiKey?: string,
+    zulipFeatureLevel?: number | null,
```

Comment:
Nits in commit message for 

f514d6a7a Account type: Add a property featureLevel.

- Let's be consistent with the name we use as an identifier in the code: `zulipFeatureLevel`. Setting it off with \`backticks\` makes it quite clear (in addition to the camelCase capitalization) that we're referring to an identifier. There may be other places where this comment applies.
- Let's specify that we're talking about `loginSuccess` and `realmAdd` in the `accountsReducer`, not the functions with the same names in `accountActions`. It could be inferred from context and by looking at the diff, but it's nice to help the reader avoid ambiguity.